db/data/regions/IT.yml

```diff
@@ -14,6 +14,7 @@ zip_regex: "^(IT?-?)?\\d{5}$"
 zip_example: '00144'
 phone_number_prefix: 39
 building_number_required: true
+building_number_may_be_in_address2: true
```

Comment:
is Italy the only country where building number is in address2 or are there others we are missing?

Reply:
Germany, Spain, and The Netherlands also allow building number in address2. For those countries, the `.yml` files already had this line.